tests/Aspire.Hosting.Tests/Orchestrator/ParameterProcessorTests.cs

```diff
@@ -801,6 +880,24 @@ public Task SaveSectionAsync(DeploymentStateSection section, CancellationToken c
         }
     }
 
+    private sealed class MockUserSecretsManager : IUserSecretsManager
```

Comment:
nit: Is there an existing MockUserSecretsManager somewhere that could be put in a shared location and reused? I've found having one good mock is better than many private mocks